installer/install-rust-analyzer.sh

```diff
@@ -13,7 +13,12 @@ darwin)
   ;;
 esac
 
-version="2021-01-18"
+if [ ! which grep &> /dev/null ] || [ ! which cut &> /dev/null ]; then
+  version="2021-02-15"
+else
+  version=$(curl -sL https://api.github.com/repos/rust-analyzer/rust-analyzer/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
+fi
+
 curl -L -o rust-analyzer-$platform "https://github.com/rust-analyzer/rust-analyzer/releases/download/$version/rust-analyzer-$platform"
```

Comment:
[IMO]
`latest` can download latest release.

e.g.
`curl -L -o "rust-analyzer-$platform" "https://github.com/rust-analyzer/rust-analyzer/releases/latest/download/rust-analyzer-$platform"`

Is this enough?

Reply:
fixed